src/Symfony/Component/Form/Extension/Validator/ViolationMapper/ViolationMapper.php

```diff
@@ -13,22 +13,33 @@
 
 use Symfony\Component\Form\FormError;
 use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormRendererInterface;
 use Symfony\Component\Form\Util\InheritDataAwareIterator;
 use Symfony\Component\PropertyAccess\PropertyPathBuilder;
 use Symfony\Component\PropertyAccess\PropertyPathIterator;
 use Symfony\Component\PropertyAccess\PropertyPathIteratorInterface;
 use Symfony\Component\Validator\ConstraintViolation;
+use Symfony\Contracts\Translation\TranslatorInterface;
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
 class ViolationMapper implements ViolationMapperInterface
 {
+    private $formRenderer;
+    private $translator;
+
     /**
      * @var bool
```

Comment:
while doing so, let's remove this annotation and make the property default to false